front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -163,13 +159,20 @@ class FullscreenModal extends PureComponent<Props, State> {
           </StyledFocusOn>
         </Container>
       </CSSTransition>,
-      document.getElementById('modal-portal') as HTMLElement
```

Comment:
This was causing the bug: type casting 🤬 This value can also be `null`. If it's `null`, this resulted in the DOM target not being available.

Reply:
This line was causing the bug. `document.getElementById('modal-portal')` can also be `null`. If it was `null`, this resulted in the DOM target not being available.